### PR TITLE
[fix #6689]: test server auth cmd `invite_1_greeter_wait_peer`

### LINKED ISSUE #6689: Test server auth cmd: `invite_1_greeter_wait_peer`

### PR DESCRIPTION
**Pull request by FirelightFlagboy**
This PR add test for the command `invite_1_greeter_wait_peer` with the failure response `invitation_not_found` and `invitation_deleted`

### CODE REVIEW

File: server/tests/api_v4/authenticated/test_invite_1_greeter_wait_peer.py [hunk 1]
```diff
@@ -1,57 +1,156 @@
...
-            await second(tg.cancel_scope)
+            await config.second(tg.cancel_scope)
 
-    assert rep == authenticated_cmds.v4.invite_1_greeter_wait_peer.RepOk(
+    assert config.rep == authenticated_cmds.v4.invite_1_greeter_wait_peer.RepOk(
         claimer_public_key=claimer_private_key.public_key
     )
+
+
+async def test_invitation_not_found(
```
Comment (mmmarcos) [inline server/tests/api_v4/authenticated/test_invite_1_greeter_wait_peer.py:112]:
```suggestion
async def test_invite_1_greeter_wait_peer_invitation_not_found(
```

File: server/tests/api_v4/authenticated/test_invite_1_greeter_wait_peer.py [hunk 2]
```diff
@@ -1,57 +1,156 @@
...
+        async with anyio.create_task_group() as tg:
+            tg.start_soon(config.first, tg.cancel_scope)
+            await spy.wait(EventEnrollmentConduit)
+
+            await config.second(tg.cancel_scope)
+
+    assert config.rep == authenticated_cmds.v4.invite_1_greeter_wait_peer.RepInvitationNotFound()
+
+
+async def test_invitation_deleted(
```
Comment (mmmarcos) [inline server/tests/api_v4/authenticated/test_invite_1_greeter_wait_peer.py:132]:
```suggestion
async def test_invite_1_greeter_wait_peer_invitation_deleted(
```

**Review by mmmarcos — APPROVED** [review]
LGTM. Just some suggestions to rename tests for consistency with existing test naming. 

BTW, maybe you can add `test_invite_1_greeter_wait_peer_enrollment_wrong_state` with a `xfail(reason="TODO:...")` mark. This would make easier to look for missing test later.

**Comment by FirelightFlagboy** [comment]
I understand the consistency issue, but the information is already present in the filename.

To run a test manually, with the currently pattern is `{path}/{invite_state}.py::test_{invite_state}_{response_type}`, I find it too lengthy

**Comment by mmmarcos** [comment]
> I understand the consistency issue, but the information is already present in the filename.

It's not really an issue, is a quality that we want to have, specially for large codebases. 
 
> To run a test manually, with the currently pattern is `{path}/{invite_state}.py::test_{invite_state}_{response_type}`, I find it too lengthy

Your argument is valid, I'm not against it. IMO for this type of changes, either we apply it everywhere (so we'll need an issue, and treat it in another PR) or nowhere. In the end, having mixed ways of doing things, makes it hard to understand and contribute.

**Pull request merged**